core/src/main/java/org/apache/iceberg/DataTableScan.java

```diff
@@ -112,4 +112,8 @@ public CloseableIterable<FileScanTask> doPlanFiles() {
 
     return manifestGroup.planFiles();
   }
+
+  public void setStats(boolean setStats) {
```

Comment:
Rather than calling this `setStats`, I think it should be `includeStats()`. No need for the boolean argument, too. It should be sufficient to call this like we do `ignoreResiduals()`.

Reply:
Fixed. Thanks